### PR TITLE
Food legend + dialog boxes + food menu item ui fix

### PR DESCRIPTION
**Pull request by sidthesquid**
- Added a food legend (info button near hours)
- Dialog boxes instead of jank fragments, menu item centered finally
- Quick map icon fix for #132

### CODE REVIEW

**Comment by rustielin** [comment]
@sidthesquid some merge conflicts after #132

**Comment by sidthesquid** [comment]
> @sidthesquid some merge conflicts after #132

Will resolve

File: app/src/main/java/com/asuc/asucmobile/domain/main/PopUpActivity.java [hunk 1]
```diff
@@ -19,9 +24,10 @@
 
 /**
  * Created by alexthomas on 4/9/17.
+ * Modified by sidthesquid 3/18
```
Comment (rustielin) [inline app/src/main/java/com/asuc/asucmobile/domain/main/PopUpActivity.java]: i c u 👀

Comment (sidthesquid) [reply app/src/main/java/com/asuc/asucmobile/domain/main/PopUpActivity.java]: HAHA shhh

File: app/src/main/java/com/asuc/asucmobile/domain/main/LegendPopUpActivity.java [hunk 2]
```diff
@@ -0,0 +1,93 @@
...
+    }
+    @Override
+    public View onCreateView(LayoutInflater inflater, ViewGroup container, Bundle savedInstanceState) {
+        super.onCreate(savedInstanceState);
+
+        View v = inflater.inflate(R.layout.fragment_food_legend, container, false);
+
+        ListView legendList = (ListView) v.findViewById(R.id.legend_list);
+        ArrayList<Pair<Drawable, String>> items = new ArrayList<>();
+        items.add(new Pair(ContextCompat.getDrawable(this.getActivity(), R.drawable.milk), "Milk"));
```
Comment (rustielin) [inline app/src/main/java/com/asuc/asucmobile/domain/main/LegendPopUpActivity.java]: Would be cleaner if you defined these food type Strings somewhere else. See `values` folder

Comment (sidthesquid) [reply app/src/main/java/com/asuc/asucmobile/domain/main/LegendPopUpActivity.java]: @rustielin good point i will get on that

File: app/src/main/java/com/asuc/asucmobile/domain/main/PopUpActivity.java [hunk 3]
```diff
@@ -73,19 +80,30 @@ protected void onCreate(@Nullable Bundle savedInstanceState) {
                 icon.setImageResource(R.drawable.sleeppod_map_icon);
                 markerTitle.setTextColor(getResources().getColor(R.color.pink));
                 break;
+            case "Ford Go Bike":
```
Comment (rustielin) [inline app/src/main/java/com/asuc/asucmobile/domain/main/PopUpActivity.java]: The strings  are defined in `values` for reusability and consistency

Comment (sidthesquid) [reply app/src/main/java/com/asuc/asucmobile/domain/main/PopUpActivity.java]: @rustielin will change this too

**Pull request merged**